### PR TITLE
refactor(components, design): Atlantis retheme

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Evolving the visual language of Atlantis through updates to the design and components packages

### Added

- Base palette tokens for a small `taupe` color range to be consumed by various semantic values

### Changed

- Design tokens changing primarily at the semantic color and border radius level, with some "base" colors updating as well

## Testing

Ensure that components look the same as the prior iteration of Atlantis with the `inRetheme` configuration enabled

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Banner/Banner.tsx [hunk 1]
```diff
@@ -103,17 +89,12 @@ export function Banner({
...
-          className={styles.closeButton}
-          onClick={handleClose}
-          aria-label="Close this notification"
-        >
-          <Icon
-            name="cross"
-            color={JOBBER_RETHEME ? "interactiveSubtle" : iconColors[type]}
+        <div className={styles.closeButton}>
+          <ButtonDismiss
+            ariaLabel={"Dismiss notification"}
```
Comment (MichaelParadis) [inline packages/components/src/Banner/Banner.tsx:94]: Just comment that this will break some JO tests but not that big of an issue

Comment (chris-at-jobber) [reply packages/components/src/Banner/Banner.tsx:94]: Honestly I'm happy to revert the ariaLabel change if it speeds things up; it's really unrelated to the retheme. Although if it's just a matter of updating the strings in related Jobber tests, I can live with that 😄

Comment (MichaelParadis) [reply packages/components/src/Banner/Banner.tsx:94]: Yeah, it is just updating strings. Super quick and not many failed when I ran it on JO. Most of the tests were snapshots. I like the update

**Review by MichaelParadis — APPROVED** [review]
LGTM!

**Pull request merged**